gratipay/testing/browser.py

```diff
@@ -82,10 +91,51 @@ def has_element(self, selector, timeout=None):
         """
         return self.is_element_present_by_css(selector, timeout)
 
+    def wait_to_disappear(self, selector, timeout=2):
+        """Wait up to ``timeout`` seconds for element specified by ``selector``
+        to disappear, returning ``None``.
+        """
+        end_time = time.time() + timeout
+        while time.time() < end_time:
+            if not self.has_element(selector):
+                return
+        raise NeverLeft()
```

Comment:
I think this could make it a bit difficult to debug failures - (What never left?). Maybe we could make it better by adding a message saying that `The element specified #{selector} did not disappear`